### PR TITLE
DSD-1775/add esc func to accordion

### PR DESCRIPTION
**Pull request by oliviawongnyc**
Fixes JIRA ticket [DSD-1775](https://jira.nypl.org/browse/DSD-1775)

## This PR does the following:

- Updates `Accordion` to close focused panel when "esc" key is pressed.

## How has this been tested?

- Locally, in Storybook

## Accessibility concerns or updates

- This is meant to be an accessibility improvement.

-

### Checklist:

- [ ] I have updated the Storybook documentation accordingly.
- [ ] I have added relevant accessibility documentation for this pull request.
- [ ] All new and existing tests passed.

### Front End Review:

- [ ] Review the Vercel preview deployment once it is ready.

### CODE REVIEW

File: src/components/Accordion/Accordion.mdx [hunk 1]
```diff
@@ -9,10 +9,10 @@ import { changelogData } from "./accordionChangelogData";
...
 # Accordion
 
-| Component Version | DS Version |
-| ----------------- | ---------- |
-| Added             | `0.1.0`    |
-| Latest            | `3.0.0`    |
+| Component Version | DS Version   |
+| ----------------- | ------------ |
+| Added             | `Prerelease` |
+| Latest            | `3.0.0`      |
```
Comment (bigfishdesign13) [inline src/components/Accordion/Accordion.mdx]: The `Prerelease` value should go in the `Latest` row and the `Added` row should be reverted to `0.1.0`.

**Review by bigfishdesign13 — APPROVED** [review]
This works perfectly, but we should also get a more thorough review of the code.

**Review by EdwinGuzman — COMMENTED** [review]
Look good. I can't see a way to implement this without `useState` which is what I would prefer. Just update what Marty pointed out.

File: src/components/Accordion/Accordion.mdx (hunk 1 above)
Comment (oliviawongnyc) [reply src/components/Accordion/Accordion.mdx]: Oops, thanks, Marty!

**Pull request merged**